src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -179,13 +179,20 @@ function useChatUtilsHook(): UseChatUtilsHook {
     navigation.goBack();
   };
   const handleTextSystem = (item): string => {
+    let message = item?.description || item?.message;
+    if (message === ' ') {
+      message = 'sent media 🎆🌆🌉';
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for an empty message only considers a single space. It might be better to trim the message and then check if it's empty, which would handle cases where the message consists of multiple spaces or other whitespace characters.
```diff
-    if (message === ' ') {
+    if (message.trim() === '') {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->